### DISCUSSION TITLE
【共学营打卡】【第4次 Milestone】发行 NFT

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#192] [Projects-Others]
> Aptos Version：https://mp.weixin.qq.com/s/Of9xGuHwcIW1CM-6MHCnyQ

任务：通过 nft_issuer 脚本在 `testnet` 发布 NFT Collection 与 NFT。
打卡内容：
1. collection 在浏览器中的地址；
2. NFT 在任一钱包插件中的截图。

**Comment by MagicGordon** [comment]
https://explorer.aptoslabs.com/token/f28aaf0e8f2d70d4f690a5e030e2e7245da8b301fa248589537460dbb84b5d4d/0
<img width="389" alt="706" src="https://user-images.githubusercontent.com/19465870/200173313-d94754a7-e9a4-4d8d-9ffa-eca8832f8cc7.png">

**Comment by zhaoge5859** [comment]
https://explorer.aptoslabs.com/token/6bd23f5825d318d42703cd07b2ec6ba40c4c118ea38c05623e45fba68ee10dc2/0
<img width="400" alt="截屏2022-11-06 22 35 12" src="https://user-images.githubusercontent.com/116735858/200177061-dda450e8-db29-4ead-8dfb-b553ce699b99.png">

**Reply by zhaoge5859** [reply]
<img width="414" alt="截屏2022-11-06 22 39 09" src="https://user-images.githubusercontent.com/116735858/200177120-c5b7383b-767f-47c3-bffb-0baed8bf9153.png">

**Comment by livelybug** [comment]
https://explorer.aptoslabs.com/token/35ad3f4df30d1c3869ab9a8df26108a94622128043d8d49aeef963deb79e82e7/0?network=testnet

![image](https://user-images.githubusercontent.com/13627060/201300470-a879e353-e470-4d54-bcd4-c9b23fd8cc20.png)

**Comment by xiegudong45** [comment]
![image](https://user-images.githubusercontent.com/30063330/201528598-dd18f7ae-539e-4146-99ad-ade6e56f302e.png)

**Reply by xiegudong45** [reply]
collection地址：https://explorer.aptoslabs.com/token/4094cd5e15979fc398c05e4bce36e6fa2413a9c1c4dcac8693558b0e7681bb9d/0?network=testnet

**Comment by v1xingyue** [comment]
collection地址： https://explorer.aptoslabs.com/token/b611f2a93b392f28b1c732fe5816a878db27c2976a69294cd36c5ea7ee82564b/0/overview

![image](https://user-images.githubusercontent.com/974169/201557861-843f9c51-602a-4cb7-904c-42504ed74b1b.png)

**Comment by baidang201** [comment]
<img width="381" alt="image" src="https://user-images.githubusercontent.com/3405789/218300177-c5133b3b-7bd7-4933-a797-b4c8a2b50f48.png">
